orm/schemas/staticsSchema.js

```diff
@@ -5,15 +5,14 @@ const utilsDatabase       = require('../../utils/database');
 const Schema              = mongoose.Schema;
 
 const StaticsSchema = new Schema({
-    code         : {type: String, required: true, unique: true},
-    type         : {type: String, required: true},
-    active       : {type: Boolean, default: false},
-    creationDate : {type: Date, default: Date.now},
-    modifyDate   : {type: Date, default: Date.now},
-    group        : {type: String, default: ''},
+    code        : {type: String, required: true, unique: true},
+    type        : {type: String, required: true},
+    active      : {type: Boolean, default: false},
+    modifyDate  : {type: Date, default: Date.now},
```

Comment:
I think "timestamps" allow to user createdAt AND modifiedAt. No needs to use "modifyDate" !?